masterlist.yaml

```
@@ -9121,6 +9121,12 @@ plugins:
   - name: 'Crossbow Integration - USSEP.esp'
     tag: [ Text ]
 
+  - name: 'OPHybrid.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/104477/'
+        name: 'Curse of the Dragonborn - REDUX'
+    group: *earlyLoadersGroup
+    
   - name: 'CutContentRestoration.esp'
```

There are still some trailing spaces here:
```suggestion
    group: *earlyLoadersGroup

  - name: 'CutContentRestoration.esp'
```

---

Whoops. Taken care of.